.github/workflows/ci.yml

```diff
@@ -25,14 +25,9 @@ on:
       - 'NOTICE'
 
 jobs:
-  wrapper-validation:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - uses: gradle/wrapper-validation-action@v1
-
   ktlint:
     runs-on: ubuntu-latest
+    if: env.ENABLE_CHECK == 'true'
```

Comment:
https://github.com/Goooler/DemoApp/actions/runs/2947525344

[Invalid workflow file: .github/workflows/ci.yml#L30](https://github.com/Goooler/DemoApp/actions/runs/2947525344/workflow)
The workflow is not valid. .github/workflows/ci.yml (Line: 30, Col: 9): Unrecognized named-value: 'env'. Located at position 1 within expression: env.ENABLE_CHECK == 'true'